src/Cake/Scripting/Roslyn/Stable/RoslynScriptSessionFactory.cs

```diff
@@ -10,15 +10,15 @@ namespace Cake.Scripting.Roslyn.Stable
 {
     using Core.IO;
```

Comment:
@mholo65 Then I think we can let it stay as is for now and perhaps look at refactoring later.